src/components/header/header.tsx

```diff
@@ -1,31 +1,167 @@
 import React from "react";
-import { View, Text, StyleSheet } from "react-native";
-
+import { View, Text, StyleSheet, TextInput, Picker, Modal, TouchableHighlight } from "react-native";
+import { Ionicons } from '@expo/vector-icons';
+import { Dropdown } from 'react-native-material-dropdown';
+//import * as Font from 'expo-font';
+ 
+ 
 export default class Header extends React.Component {
     
+  
+    state = {user: 'asia', modalVisible: false}
+    setModalVisible(visible) {
+        this.setState({modalVisible: visible});
+    }
+    //Trying to get the fonts to work
+    /*async componentDidMount() {
+    await Font.loadAsync({
+      'montserrat-medium': require('../../assets/fonts/Montserrat-Medium.ttf'),
+    });
+    this.setState({ fontLoaded: true });
+  }*/
     render() {
-        return (
-            <View style={styles.header}>
```

Comment:
Noticed that with a screen that has a notch (Tested with Pixel 3 XL Emulator), the logo is a little bit too close to the notch. Add a little padding, or replace static padding with one that takes android notch into account.